echo.go

```diff
@@ -769,7 +769,7 @@ func (e *Echo) configureServer(s *http.Server) (err error) {
 			}
 		}
 		if !e.HidePort {
-			e.colorer.Printf("⇨ http server started on %s\n", e.colorer.Green(e.Listener.Addr()))
+			e.Logger.Infof("⇨ http server started on %s\n", e.colorer.Green(e.Listener.Addr()))
```

Comment:
One side effect of doing this is that you get the color characters on the logs

```
{"time":"2021-07-10T00:09:56.928808-03:00","level":"INFO","prefix":"echo","file":"echo.go","line":"772","message":"⇨ http server started on \x1b[32m[::]:1323\x1b[0m\n"}
```

Maybe the color could be removed.

The other side effect is that in order to see this you need to set echo.Debug to true

Reply:
@pafuent Thanks for the answer.

#### Color
Should I remove `e.colorer.Green`?

#### Debug/Logging

In my opinion, it is somehow a logging message.

What do you think in general about this? A good idea? or something to drop?